src/sentry/discover/models.py

```diff
@@ -76,6 +85,51 @@ def set_projects(self, project_ids):
             )
 
 
+class TeamKeyTransactionModelManager(BaseManager):
+    @staticmethod
+    def __schedule_invalidate_project_config_transaction_commit(instance, trigger):
+        project = getattr(instance.project_team, "project")
```

Comment:
I'm confused about this: `getattr` with two arguments will raise an `AttributeError` if `project` does not exist, so we might as well write `instance.project_team.project`. Unless we do not want it to raise an exception, in which case we should use `getattr(a, b, None)`.

https://docs.python.org/3/library/functions.html#getattr

Reply:
oh you are right!
```
In [1]: class Foo:
   ...:     pass
   ...:

In [2]: f = Foo()

In [3]: getattr(f, "porject")
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
Input In [3], in <cell line: 1>()
----> 1 getattr(f, "porject")

AttributeError: 'Foo' object has no attribute 'porject'

In [4]: getattr(f, "porject", None)

In [5]:
```

Reply:
@jjbayer fixed!